src/BuildingRegistry.Api.BackOffice/BuildingUnit/BuildingUnitController-Plan.cs

```diff
@@ -61,11 +62,11 @@ public partial class BuildingUnitController
             }
             catch (AggregateIdIsNotFoundException)
             {
-                throw new ApiException(ValidationErrorMessages.BuildingUnit.BuildingUnitNotFound, StatusCodes.Status404NotFound);
+                throw new ApiException(ValidationErrors.Common.BuildingUnitNotFound.Message, StatusCodes.Status404NotFound);
```

Comment:
Should we use the error message BuildingUnitNotFound when planning a building unit?

Reply:
No since `BuildingId` is given in the request BuildingNotFound can be used, although not as a 404  but a 400.